packages/expo-constants/src/Constants.types.ts

```diff
@@ -1,36 +1,76 @@
 import { ExpoConfig } from '@expo/config-types';
 
+// @needsAudit
 export enum AppOwnership {
+  /**
+   * It is a [standalone app](../../../distribution/building-standalone-apps#building-standalone-apps).
+   */
   Standalone = 'standalone',
+  /**
+   * The experience is running inside of the Expo Go app.
+   */
   Expo = 'expo',
+  /**
+   * It has been opened through a link from a standalone app.
+   */
   Guest = 'guest',
 }
 
+// @docsMissing
 export enum ExecutionEnvironment {
   Bare = 'bare',
   Standalone = 'standalone',
   StoreClient = 'storeClient',
 }
 
+// @needsAudit
+/**
+ * Current supported values are `handset` and `tablet`. Apple TV and CarPlay will show up
+ * as `unsupported`.
+ */
 export enum UserInterfaceIdiom {
   Handset = 'handset',
   Tablet = 'tablet',
   Unsupported = 'unsupported',
 }
 
+// @needsAudit
 export interface IOSManifest {
+  /**
+   * The build number specified in the embedded **Info.plist** value for `CFBundleVersion` in this app.
+   * In a standalone app, you can set this with the `ios.buildNumber` value in **app.json**. This
+   * may differ from the value in `Constants.manifest.ios.buildNumber` because the manifest
+   * can be updated, whereas this value will never change for a given native binary.
+   * The value is set to `null` in case you run your app in Expo Go.
+   */
   buildNumber: string;
+  /**
+   * The Apple internal model identifier for this device, e.g. `iPhone1,1`.
+   */
   platform: string;
   /**
-   * @deprecated Moved to `expo-device` - `Device.modelName`
+   * The human-readable model name of this device, e.g. `"iPhone 7 Plus"` if it can be determined,
+   * otherwise will be `null`.
+   * @deprecated Deprecated. Moved to `expo-device` as [`Device.modelName`](../device/#devicemodelname).
    */
   model: string | null;
+  /**
+   * The user interface idiom of this device, i.e. whether the app is running on an iPhone or an iPad.
+   */
   userInterfaceIdiom: UserInterfaceIdiom;
+  /**
+   * The version of iOS running on this device, e.g. `10.3`.
+   */
   systemVersion: string;
   [key: string]: any;
 }
 
+// @needsAudit
 export interface AndroidManifest {
+  /**
+   * The version code set by `android.versionCode` in app.json.
+   * The value is set to `null` in case you run your app in Expo Go.
+   */
```

Comment:
```suggestion
  /**
   * The version code set by `android.versionCode` in app.json.
   * The value is set to `null` in case you run your app in Expo Go.
   * @deprecated Deprecated. Use `expo-application`'s [`Application.nativeBuildVersion`](../application/#applicationnativebuildversion).
   */
```